libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m

```diff
@@ -1421,25 +1421,6 @@ - (void)testBlockUpdate {
                              [self.currentExpectation fulfill];
                          }];
     [self waitForExpectation];
-    self.currentExpectation = [self expectationWithDescription:@"performUpsertWithObjectType-upserting contact"];
-    fields[LAST_NAME] = lastName;
-    [api performUpsertWithObjectType:CONTACT
```

Comment:
That upserts works with an admin user but gives a 400 "insufficient access rights on cross-reference id" error when running with a standard user.